src/lsp/cobol_parser/text_lexer.ml

```diff
@@ -24,12 +24,24 @@ module TYPES = struct
       mutable reserved: bool;
       mutable enabled: bool;
     }
-  and token_handle = optional_token
+  and keyword_handle = optional_token
 
+  type intrinsic_handle =
+    {
+      token: Grammar_tokens.token;
+      mutable specified: bool;
```

Comment:
What do you mean by `specified`? Is it "registered", without need to `FUNCTION` prefix?